src/MakeJsonCommand.php

```diff
@@ -63,6 +66,7 @@ class MakeJsonCommand extends WP_CLI_Command {
 	public function __invoke( $args, $assoc_args ) {
 		$purge           = Utils\get_flag_value( $assoc_args, 'purge', true );
 		$update_mo_files = Utils\get_flag_value( $assoc_args, 'update-mo-files', true );
+		$map_paths       = Utils\get_flag_value( $assoc_args, 'use-map', false );
```

Comment:
```suggestion
		$map_paths       = Utils\get_flag_value( $assoc_args, 'use-map', '' );
```

An empty string as the default value could make `build_map()` a bit simpler

Reply:
It would, but then I couldn't differentiate between argument not given and `--use-map=`

Reply:
Is such differentiation important?

Reply:
I think I was being unclear: We couldn't know whether the argument was given or not. Since we only want to transform the sources if the argument is given, it is.